src/com/epam/javacourse/homework4/Shrub.java

```diff
@@ -0,0 +1,34 @@
+package com.epam.javacourse.homework4;
+
+public class Shrub extends Plant {
+    private final int height;
+
+    public Shrub(String name, String type, String nativeRegion, int height) {
+        super(name, type, nativeRegion);
+        this.height = height;
+    }
+
+    @Override
+    public String getPlantType() {
+        return "Shrub";
+    }
+
+    @Override
+    public String getNativeRegion() {
+        return super.getNativeRegion();
+    }
+
+    @Override
+    public String getPlantName() {
+        return super.getPlantName();
+    }
+
+    private int getHeight() {
```

Comment:
Information about height is required but never used.
In this case, you can override `toString` method from the `Plant` class and add this specific for the `Shrub` information in it.

Reply:
done